ramalama/engine.py

```diff
@@ -90,11 +92,30 @@ def add_privileged_options(self):
                 ]
 
     def add_subcommand_env(self):
+        """
+        - Uses the LLAMA_PROMPT_PREFIX environment variable if defined (must be ≤ 50 characters).
+        - Falls back to a Podman-specific emoji (🦭) or Docker-specific emoji (🐋) and if not set 🦙.
+        - Appends the appropriate '--env' argument to self.exec_args.
+
+        Raises:
+            RuntimeError: If the environment-defined LLAMA_PROMPT_PREFIX exceeds MAX_PROMPT_PREFIX_LEN.
+        """
+        prefix = os.environ.get("LLAMA_PROMPT_PREFIX")
+
+        if prefix:
+            if len(prefix) > MAX_PROMPT_PREFIX_LEN:
```

Comment:
This is overengineered, it's not uncommon to have prompts over 50 characters:

```
ecurtin@ecur arm64 13:21:09 1 use-wrapper-everywhere 0<>0
~/git/ramalama$
```

Reply:
simple or remove the verification or increase the max. From my tests, when flooding the prompt I got stuck.